content/en/docs/reference/kubectl/kubectl.md

```diff
@@ -369,6 +369,14 @@ kubectl [flags]
 </td>
 </tr>
 
+<tr>
+<td colspan="2">KUBECTL_INTERACTIVE_DELETE</td>
+</tr>
+<tr>
+<td></td><td style="line-height: 130%; word-wrap: break-word;">When set to true, the --interactive flag in the kubectl delete command will be activated, allowing users to preview and confirm resources before proceeding to delete by passing this flag.
```

Comment:
<td></td><td style="line-height: 130%; word-wrap: break-word;">When set to true, the <tt>--interactive</tt> flag in the kubectl delete command will be activated, allowing users to preview and confirm resources before proceeding to delete by passing this flag.